src/locales/en/common.json

```diff
@@ -1076,8 +1080,8 @@
 
   "walletConnectContent": {
     "title": {
-      "walletConnectRequests": "WalletConnect Request",
-      "walletConnect": "WalletConnect"
+      "walletConnectRequests": "Connect to {{app}}",
+      "description": "Be a DeFi chef with Sushi. Swap, earn, stack yields, lend, borrow, leverage all on one decentalized, community driven platform"
```

Comment:
This description is Sushi Swap specific. We probably should display description as provided but wallet connect request. Pls remove this entry.